packages/veritone-react-common/src/components/SimpleSearchBar/index.js

```diff
@@ -0,0 +1,227 @@
+import React, { Fragment } from 'react';
+import cx from 'classnames';
+import { bool, func, arrayOf, any, string } from 'prop-types';
+import {
+  IconButton,
+  Paper,
+  Divider,
+  CircularProgress,
+  Zoom,
+  ClickAwayListener,
+  Input,
+  MenuItem
+} from '@material-ui/core';
+import { Search, Clear } from '@material-ui/icons';
+
+import styles from './styles.scss';
+
+// eslint-disable-next-line
+const Div = ({ className, children }) => (
+  <div className={className}>{children}</div>
+);
+
+export class SimpleSearchBarBase extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  state = {
+    renderKey: 101,
+    isLoading: true,
+    focused: false
+  };
+
+  clickAway = () => {
+    if (this.props.resetOnClickAway) {
+      this.props.onChange('');
```

Comment:
Lines 36-40 should just be `this.clear()` to make sure all reset logic is done consistently?